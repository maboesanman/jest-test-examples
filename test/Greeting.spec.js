const shallowMount = require('@vue/test-utils').shallowMount;
const Greeting = require('../src/Greeting.vue');

let component;
beforeEach(() => {
  component = shallowMount(Greeting, {
    propsData: { name: 'Mason' }
  })
})

describe('Greeting', () => {
  it('renders correctly', () => {
    expect(component.element).toMatchSnapshot();
  });
});