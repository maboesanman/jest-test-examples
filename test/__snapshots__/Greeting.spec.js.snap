// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting renders correctly 1`] = `
<p>
  Hello Mason
</p>
`;
